go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/kubernetes-sigs/minibroker
+
+go 1.13
+
+require (
+	github.com/Masterminds/semver v1.4.0
+	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
+	github.com/davecgh/go-spew v1.1.1
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/protobuf v1.4.1 // indirect
+	github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c // indirect
+	github.com/googleapis/gnostic v0.1.0 // indirect
+	github.com/gorilla/context v0.0.0-20160226214623-1ea25387ff6f // indirect
+	github.com/gorilla/mux v1.6.1 // indirect
+	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
+	github.com/juju/ratelimit v1.0.1 // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/onsi/ginkgo v1.12.0 // indirect
+	github.com/onsi/gomega v1.10.0 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/pmorie/go-open-service-broker-client v0.0.0-20180304212357-e8aa16c90363
+	github.com/pmorie/osb-broker-lib v0.0.0-20180516212803-87d71cfbf342
+	github.com/prometheus/client_golang v1.6.0
```

Comment:
v0.8.0 → v1.6.0 (I assume to pick up go modules compatibility or something?)

Reply:
Yes, this one I explicitly bumped to pick up the go modules compatibility.